king_phisher/server/database/models.py

```diff
@@ -397,6 +398,95 @@ class Message(Base):
 	credentials = sqlalchemy.orm.relationship('Credential', backref='message', cascade='all, delete-orphan')
 	visits = sqlalchemy.orm.relationship('Visit', backref='message', cascade='all, delete-orphan')
 
+@register_table
+class ModuleQuestion(Base):
+	__repr_attributes__ = ('question',)
+	__tablename__ = 'module_questions'
+	id = sqlalchemy.Column(sqlalchemy.Integer, primary_key=True)
+	question = sqlalchemy.Column(sqlalchemy.String, nullable=False)
```

Comment:
This object should also have a nullable description string field for internal notes regarding the question.